src/servises/expensesServises.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+let expenses = [];
+
+function getNewId() {
+  const maxId = Math.max(...expenses.map(el => el.id));
+
+  if (maxId) {
+    return 1;
+  }
+
+  return maxId + 1;
+}
```

Comment:
is better to use ternary operator here

Reply:
I've tried... I don't understand why it doesn't work(((